packages/react-dom/src/events/getEventTarget.js

```diff
@@ -15,7 +15,9 @@ import {TEXT_NODE} from '../shared/HTMLNodeType';
  * @return {DOMEventTarget} Target node.
  */
 function getEventTarget(nativeEvent) {
-  let target = nativeEvent.target || window;
+  // Fallback to nativeEvent.srcElement for IE9
+  // https://github.com/facebook/react/issues/12506
+  let target = nativeEvent.target || nativeEvent.srcElement || window;
```

Comment:
Should we have an internal unit test for this?

Reply:
I'd really like to track down the reason for this so we don't accidentally remove it again as well. Everything I can find suggests that `srcElement` should _not_ be needed in IE9, so I'm wondering where the issue is coming from. I haven't found much some-places suggest that when `attachEvent` is used it follows the ie8 event model (with target undefined), I wonder if some branches are still hitting attachEvent?

Reply:
the change polyfill uses it, Perhaps that's the root cause? I think we could use `addEventListener` there...

Reply:
Interesting... I actually remove that listener in #12505. I'll hunt!